### PR TITLE
States

### PR DESCRIPTION
**Pull request by morvaygergo**

### CODE REVIEW

File: components/App.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+import Button from "./Button";
+import Display from "./Display";
+
+class App extends Component {
+  state = {
+    counter: 0
+  };
+
+  incrementCounter = () => {
+    let { counter } = this.state;
```
Comment (laurakocsis) [inline components/App.js]: const should be used instead of let

File: components/App.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    this.setState({ counter: ++counter });
+  };
+
+  decreaseCounter = () => {
+    let { counter } = this.state;
```
Comment (laurakocsis) [inline components/App.js]: const should be used instead of let

File: components/App.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    if (counter > 0) {
+      this.setState({ counter: --counter });
+    }
+  };
+
+  componentDidMount() {
```
Comment (laurakocsis) [inline components/App.js:10]: styleguide issue: componentDidMount should be the first method after state

File: components/App.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+      if (e.keyCode === 38) {
+        this.incrementCounter();
+      } else if (e.keyCode === 40) {
+        this.decreaseCounter();
+      }
+    });
+  }
+
+  render() {
+    let { counter } = this.state;
```
Comment (laurakocsis) [inline components/App.js]: const should be used instead of let

**Review by laurakocsis — CHANGES_REQUESTED** [review]
general styleguide issue: single quotes shb used instead of double quotes

**Review by laurakocsis — APPROVED** [review]

**Review by gyorkibence — APPROVED** [review]

**Pull request merged**